# 2024 Bioinformatics methods for nanopore sequencing analysis in infection research

A practical introduction to long-read bioinformatics in infection research. Here, we focus on (bacterial) genome reconstruction.

## Instructor

* [Martin HÃ¶lzer](https://hoelzer.github.io)

## Part I 

* [Linux re-cap](day01-tuesday/linux.md) 
* [Nanopore introduction I (intro)](day01-tuesday/nanopore.md) 
* [Nanopore introduction II (file formats & QC)](day01-tuesday/nanopore.md) 
* [Hands-on & demo](day01-tuesday/hands-on.md) 

## Part II

* [_De novo_ Assembly](day02-wednesday/assembly.md) 
* [Mapping & Visualization](day02-wednesday/mapping.md)
* [Hands-on & demo](day02-wednesday/hands-on.md) 

## Part III

* [Assembly polishing](day03-thursday/polishing.md) 
* [Variant calling](day03-thursday/variant-calling.md) 
* [Hands-on & demo](day03-thursday/hands-on.md) 

## Part IV

* [Miscellaneous](day04-friday/README.md) 
* [Hands-on & demo](day04-friday/hands-on.md) 

## Acknowledgement

This course material is partly based on the following resources and on contributions from great people:

* Sebastian "Raverjay" Krautwurst, FSU Jena, some Linux and ONT content
* Workshop structure inspired by [https://github.com/cinemaparis/2023](https://github.com/cinemaparis/2023)
* Some ONT intro slides from Josh Quick, [original](https://github.com/cinemaparis/2023/blob/main/day1-Tuesday/slides-Quick.pdf)
* Nanopore long-read bioinformatics tutorial from [timkahlke](https://timkahlke.github.io/LongRead_tutorials)

**Also, this course was prepared before for the [MISSION Kenya project at RKI](https://github.com/rki-mf1/2023-Workshop-MISSIoN-Nanopore)**